src/editortoolkit.cpp

```diff
@@ -924,9 +932,101 @@ bool EditorToolkit::Ungroup(std::string groupType, std::vector<std::string> elem
     return true;
 }
 
+bool EditorToolkit::ChangeGroup(std::string elementId, std::string contour)
+{
+    m_editInfo = "";
+    //Check if you can get drawing page
+    if(!m_doc->GetDrawingPage()) {
+        LogError("Could not get the drawing page.");
+        return false;
+    }
+    Neume *el = dynamic_cast<Neume *> (m_doc->GetDrawingPage()->FindChildByUuid(elementId));
```

Comment:
Check to make sure this is not null. If the element does not exist then `el` will be set to `nullptr` and will cause a segmentation fault when trying to dereference it in a few lines.

Reply:
fixed in fbe8061